src/ast/rewriter/bit_blaster/bit_blaster_tpl_def.h

```diff
@@ -1247,20 +1253,16 @@ void bit_blaster_tpl<Cfg>::mk_const_case_multiplier(bool is_a, unsigned i, unsig
 
 template<typename Cfg>
 bool bit_blaster_tpl<Cfg>::mk_const_multiplier(unsigned sz, expr * const * a_bits, expr * const * b_bits, expr_ref_vector & out_bits) {
+    if (!m_use_bcm) {
```

Comment:
I removed m_use_bcm. It would always be false, but my change almost for sure changes the behavior which I would have to revise or check.